### DISCUSSION TITLE
home-assistant.log

### DISCUSSION THREAD

**Discussion by nikpait** [indomus/forum#1964] [HUB: Home Assistant]
Ciao a tutti,
volevo sapere se vi è una procedura per fare in modo che l'home-assistant.log una volta raggiunta una data dimensione, venga fatto il purge.
Al momento so che questo viene fatto ogni volta che riavvio il raspberry, ma ultimamente nel giro di 1 settimana ragigunge dimensioni di 10/11 GB saturando la SD.

Grazie

Home Assistant 2022.9.6
Supervisor 2022.09.1
Operating System 9.0
Frontend 20220907.2 - latest

**Comment by snakuzzo** [comment]
Non mi pare ci sia qualcosa di automatico.
Se usassi Core (in venv o docker non fa differenza) potresti usare `--log-rotate` per cancellare le entry nel log più vecchie di `n` giorni.
Visto che usi HA OS, volendo potresti fare come indicato [QUI](https://community.home-assistant.io/t/option-to-purge-the-log/24458/16?u=snakuzzo)

**Reply by nikpait** [reply]
Grazie, provo così
